src/winetricks

```diff
@@ -15322,6 +15344,22 @@ load_ie8()
         "${WINE}" regsvr32 /i ${i} > /dev/null 2>&1
     done
 
+    if [ "${W_ARCH}" = "win64" ]; then
+        w_try_cd "${W_SYSTEM64_DLLS}"
```

Comment:
Same here.

Reply:
Done.

I also made use of the w_try_regsvr and w_try_regsvr64 helpers, and had to "whittle" down that list of dlls some due to errors, I'll document my process below.